### DISCUSSION TITLE
Unzipping using linux UI does not set executable bits required

### DISCUSSION THREAD

**Discussion by BluePython339** [Vector35/binaryninja-api#2091] [General] [answered]
Hey, the personal installation is erroring after I gave it my licence file.:

error:
LaunchProcess: failed to execvp:
/home/bluepython339/tools/binaryninja/QtWebEngineProcess
Trace/breakpoint trap (core dumped)

system:
fedora 32 5.6.14

**Comment by psifertex** [comment, marked as answer]
Use the command-line to unzip the zip file. Unfortunately some GUI tools to unzip files on linux do not maintain the +x executable bit that several files need. 

Also, this really isn't a discussion but an issue, for future problems, file issues under https://github.com/vector35/binaryninja-api/issues

**Comment by jeffli678** [comment]
late to the party. I have fixed this recently. It is not really our fault -- I just made our zip archive more robust to the different ways to extract it.
